simtools/utils/general.py

```diff
@@ -808,3 +809,59 @@ def sort_arrays(*args):
         _, value = zip(*sorted(zip(order_array, arg)))
         new_args.append(list(value))
     return new_args
+
+
+def extract_type_of_value(value) -> str:
```

Comment:
Do we need this as well for astropy Quantities? Do we want to keep them in the long style?

```
>>> type(3.*u.m)
<class 'astropy.units.quantity.Quantity'>
>>> type(u.m)
<class 'astropy.units.core.IrreducibleUnit'>
```

Or is the class always connect to get_value_unit_type which takes Quantities apart? If yes, maybe add a remark in the docstring.

Reply:
I don't see how it would be useful for astropy quantities. This is really only useful in order to save in the DB the type in a way that `type()` can then be applied directly.
It doesn't have to be related to `get_value_unit_type`, so a comment in the docstring is probably not necessary. However, I could add some sort of "protection" that if a Quantity is provided, it raises an error. What do you think?

Reply:
This would be fine with me - but you will have to judge if this is useful. If not, just leave it and merge.

Reply:
OK, I added a not implemented error for astropy units. Since tests passed before on Github and now they passed again locally, I won't wait for them to pass and just merge.